superset-frontend/src/features/tags/tags.ts

```diff
@@ -47,10 +47,15 @@ const map_object_type_to_id = (objectType: string) => {
 };
 
 export function fetchAllTags(
+  // fetch all tags (excluding system tags)
   callback: (json: JsonObject) => void,
   error: (response: Response) => void,
 ) {
-  SupersetClient.get({ endpoint: `/api/v1/tag` })
+  SupersetClient.get({
+    endpoint: `/api/v1/tag/?q=${rison.encode({
+      filters: [{ col: 'type', opr: 'custom_tag', value: true }],
```

Comment:
@Vitor-Avila you mention in your PR that you're excluding system tags yet here you're including only custom tags. Could these differ?

Reply:
@john-bodley It should be the same thing. I implemented the `custom_tag` filter operator in [this PR](https://github.com/apache/superset/pull/26701/files#diff-8db349e723a165326c5a22dd71dd76e16dafa97288e38318b16dd0c50e867eb2R34-R36) which when set to `True` returns only tags with `type == 'custom'` (created by users):
``` python
class UserCreatedTagTypeFilter(BaseFilter):  # pylint: disable=too-few-public-methods
    """
    Filter for tag type.
    When set to True, only user-created tags are returned.
    When set to False, only system tags are returned.
    """

    name = _("Is custom tag")
    arg_name = "custom_tag"

    def apply(self, query: Query, value: bool) -> Query:
        if value:
            return query.filter(Tag.type == TagType.custom)
        if value is False:
            return query.filter(Tag.type != TagType.custom)
        return query
```

[Reference](https://github.com/apache/superset/blob/master/superset/tags/models.py#L60-L76):
``` python
class TagType(enum.Enum):
    """
    Types for tags.

    Objects (queries, charts, dashboards, and datasets) will have with implicit tags based
    on metadata: types, owners and who favorited them. This way, user "alice"
    can find all their objects by querying for the tag `owner:alice`.
    """

    # pylint: disable=invalid-name
    # explicit tags, added manually by the owner
    custom = 1

    # implicit tags, generated automatically
    type = 2
    owner = 3
    favorited_by = 4
```